src/app/containers/FrontPageMain/index.jsx

```diff
@@ -2,15 +2,49 @@
 import React, { useContext } from 'react';
 import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
 import path from 'ramda/src/path';
+import Grid, { FrontPageGrid } from '#app/components/Grid';
 import { frontPageDataPropTypes } from '#models/propTypes/frontPage';
-import { Grid, GridItemConstrainedLargeWithTopMargin } from '#lib/styledGrid';
 import { ServiceContext } from '#contexts/ServiceContext';
 import FrontPageSection from '../FrontPageSection';
 import MetadataContainer from '../Metadata';
 import LinkedData from '../LinkedData';
 import ATIAnalytics from '../ATIAnalytics';
 import ChartbeatAnalytics from '../ChartbeatAnalytics';
 
+const mainColumns = {
+  group0: 6,
+  group1: 6,
+  group2: 6,
+  group3: 6,
+  group4: 8,
+  group5: 20,
```

Comment:
On the current implementation there are 22 columns for `group5`. 
```css
grid-template-columns: 1fr repeat(20, minmax(0px, 2.95rem)) 1fr;
```
This is different from what `psammead-grid` is producing. 
```css
grid-template-columns: repeat(20,1fr);
```
The diff in image sizes could be due to this. Just wanted to point this out.

Reply:
Looking at the designs  (https://app.zeplin.io/project/5c8a5e021d454124ef90669f/screen/5d80cdf1b6cd63785624f1b5), the grid is supposed to have 20 columns for group5. With the new implementation, we're using `startOffsets` and `margins` to space the first column from the page margins.

Reply:
@sadickisaac The previous implementation had the 20 grid columns for content, plus two for the spacing either side to handle the fixed-width implementation for group 5 to limit the width to 1280px. This implementation handles the spacing in a different way, by adding a `max-width` on the Grid.